lib/tasks/csv.rake

```diff
@@ -0,0 +1,42 @@
+namespace :csv do
+
+  task :certify_existing_users => :environment do
+
+    test_spreadsheet = Rails.root.join('test','lib','assets','TRAINING_DATABASE.csv')
+
+    CSV.foreach(test_spreadsheet) do |row|
+      
+      student_id = row[2]
+      name = row[3]
+      email = row[6]
+      training = row[10]
+      trainer = row[13]
+      course = row[14]
+
+      user = User.find_by(email: email)
+
+      if user && training && trainer
+        certs = user.certifications.map(&:training)
+        unless certs.include?(training)
```

Comment:
Is this going to work? Won't this return Training objects where as the training you have here is just the name?

Reply:
this works because the trianing i have isn't an objects. It's just the name obtained through the certification's training_session training name